pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,12 +1,13 @@
-# 5.9.0 
+
```

Comment:
Was the version number removed here? I looks like maybe we mean for these bullet points to be added to the 5.8.3 entry?

Reply:
I moved the bullets to other changelog entry